swagger.yaml

```diff
@@ -336,3 +422,45 @@ definitions:
           - example:
               id: "E12000001"
               label: "North East"
+  GetDimensionsResponse:
+    description: "The response body containing all available non-geography base variables for a given dataset"
+    properties:
+      area-type:
+        allOf:
+          - $ref: '#/definitions/Dimension'
+          - example:
+              id: "accommodation_type"
+              label: "Accommodation Type"   
+  Dimension:
```

Comment:
Could be wrong but I think the indentation is off here.

Reply:
Indentation updated as the GetDimensionsResponse was removed as it's no longer needed.